api/controllers/PostController.js

```diff
@@ -392,6 +421,19 @@ module.exports = {
       }
     })
     .then(() => res.ok({}), res.serverError)
-  }
+  },
 
+  checkFreshnessForCommunity: createCheckFreshnessAction(queryForCommunity, 'posts'),
```

Comment:
can you iterate through the different types here and programmatically generate both the `find` and `freshness` methods? it would be way more concise